ksqldb-metastore/src/test/java/io/confluent/ksql/metastore/MetaStoreImplTest.java

```diff
@@ -359,82 +309,11 @@ public void shouldBeThreadSafe() {
           when(source.getName()).thenReturn(SourceName.of("source" + idx));
           metaStore.putSource(source, false);
           metaStore.getSource(source.getName());
-
           metaStore.getAllDataSources();
-
-          final String queryId = "query" + idx;
-          metaStore.updateForPersistentQuery(
-              queryId,
-              ImmutableSet.of(source.getName()),
-              ImmutableSet.of(source.getName()));
-
-          metaStore.getQueriesWithSource(source.getName());
-          metaStore.getQueriesWithSink(source.getName());
-
           metaStore.copy();
-
-          metaStore.removePersistentQuery(queryId);
           metaStore.deleteSource(source.getName());
         });
 
     assertThat(metaStore.getAllDataSources().keySet(), is(empty()));
   }
-
-  @Test(timeout = 10_000)
-  public void shouldBeThreadSafeAroundRefIntegrity() throws Exception {
```

Comment:
why are we deleting this test? we haven't made the class not thread safe?

Reply:
That test is verifying queries weren't altered inside the metastore. The metastore doesn't work with queries anymore, so I removed it. It was useless.